librz/core/cmd_descs.h

```diff
@@ -55,6 +55,7 @@ RZ_IPI int rz_cmd_bsize(void *data, const char *input);
 RZ_IPI int rz_cmd_cmp(void *data, const char *input);
 RZ_IPI int rz_cmd_meta(void *data, const char *input);
 RZ_IPI int rz_cmd_debug(void *data, const char *input);
+RZ_IPI void rz_debug_continue_oldhandler(void *data, const char *input);
```

Comment:
This is an auto-generated file, as the header on top says. You should modify `cmd_descs.yaml`.
In particular, find the section for `d` command and add something like:
```yaml
  subcommands:
    - name: dc
      cname: debug_continue_oldhandler
      summary: Continue execution
      type: RZ_CMD_DESC_TYPE_OLDINPUT
```

Then when you re-run `ninja -C build` it will update both `cmd_descs.h` and `cmd_descs.c`. Make sure to have `PyYAML` python module installed.

Reply:
Sorry about that, I'll fix that.

Reply:
No problem ;)

Reply:
How/what should the function return? Since, an `RZ_IPI` returning an `int` is what the `.yaml` generated.

```
		eprintf("Cannot continue, run ood?\n");
		return false;
	}
	rz_debug_continue(core->dbg);
	return true;
}
```
Would that do?

Reply:
return a number `>= 0` when everything goes fine, `-1` when there are errors.
So it should become:
```
                 eprintf("Cannot continue, run ood?\n");
		return -1;
	}
	rz_debug_continue(core->dbg);
	return 0;
}
```

Reply:
Did that. But the CI fails somehow.

I think we have to make this available/included to `panels.c` to fix the build.

And the broken tests, too.